src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -3957,14 +3961,14 @@ class StubGenerator: public StubCodeGenerator {
   void m5_FF_GG_HH_II_epilogue(BufRegCache& reg_cache,
                                Register a, Register b, Register c, Register d,
                                int k, int s, int t,
-                               Register rtmp1, Register rtmp2, Register rmask32) {
-    // rtmp1 = rtmp1 + x + ac
-    reg_cache.get_u32(rtmp2, k, rmask32);
-    __ addw(rtmp1, rtmp1, rtmp2);
-    __ mv(rtmp2, t);
-    __ addw(rtmp1, rtmp1, rtmp2);
-
-    // a += rtmp1 + x + ac
+                               Register rtmp1, Register rtmp2, Register rtmp3) {
```

Comment:
Since `rtmp2` parameter is now not used, why not remove it?

Reply:
Good catch, I missed that. Removed.